platform/mbed_toolchain.h

```diff
@@ -376,13 +376,27 @@
 #endif
 #endif
 
+// Macro containing the filename part of the value of __FILE__. Defined as
+// string literal.
+#ifndef FILENAME
+#if defined(__CC_ARM)
+#define FILENAME __MODULE__
```

Comment:
should it follow the same prefix `MBED_FILENAME`?

Reply:
yup, fixed